app/Console/Commands/UserCreate.php

```diff
@@ -71,7 +71,7 @@ public function handle()
 
         $email = $this->ask('Email');
```

Comment:
I'd suggest to make the conversion here to keep the separation of functionalities cleaner. We never need a case-sensitive email.

```suggestion
        $email = strtolower(trim($this->ask('Email')));
```

Reply:
An email address absolutely can be case sensitive. Only the domain must be casu-insensitive